packages/protocol-kit/src/contracts/AbiType/SafeProxyFactory/v1.4.1/SafeProxyFactoryContract_v1_4_1.ts

```diff
@@ -0,0 +1,44 @@
+import { narrow } from 'abitype'
+import safeProxyFactory_1_4_1_ContractArtifacts from '@safe-global/protocol-kit/contracts/AbiType/assets/SafeProxyFactory/v1.4.1/safe_proxy_factory'
+import SafeBaseProxyFactoryContract, {
+  ProxyFactoryContractReadFunctions,
+  ProxyFactoryContractWriteFunctions
+} from '../safeProxyFactoryContract'
+
+const safeProxyFactoryContract_v1_4_1_AbiTypes = narrow(
+  safeProxyFactory_1_4_1_ContractArtifacts.abi
+)
+
+/**
+ * Represents the ABI of the Safe Proxy Factory contract version 1.4.1.
+ *
+ * @type {SafeProxyFactoryContract_v1_4_1_Abi}
+ */
+export type SafeProxyFactoryContract_v1_4_1_Abi = typeof safeProxyFactoryContract_v1_4_1_AbiTypes
+
+/**
+ * Extracts the names of read-only functions (view or pure) specific to the Safe Proxy Factory contract version 1.4.1.
+ *
+ * @type {ProxyFactory_v1_4_1_Read_Functions}
+ */
+export type ProxyFactory_v1_4_1_Read_Functions =
```

Comment:
Sometimes is prefixed with Safe and sometimes not. I would choose one for all the occurrences. Or ProxyFactory or SafeProxyFactory and stick with it through all the code